test_parser.py

```diff
@@ -0,0 +1,46 @@
+import time
+
+import arrow
+import click
+
+try:
+    from parsers.read_parser_config import parser_key_to_dict
+except Exception:
```

Comment:
ImportError is more specific.

Reply:
:+1:

Reply:
I meant in the except clause, raising the ValueError is good. e.g.
```python
 try:
     from parsers.read_parser_config import parser_key_to_dict
 except ImportError:
# raise your ValueError here
```